tests/server/rtspd.c

```diff
@@ -248,8 +247,8 @@ static int rtspd_ProcessRequest(struct rtspd_httprequest *req)
       else
         req->partno = 0;
 
-      msnprintf(logbuf, sizeof(logbuf), "Requested test number %ld part %ld",
-                req->testno, req->partno);
+      snprintf(logbuf, sizeof(logbuf), "Requested test number %ld part %ld",
+               req->testno, req->partno);
       logmsg("%s", logbuf);
```

Comment:
```suggestion
      logmsg("Requested test number %ld part %ld", req->testno, req->partno);
```
make it a `logmsg()`?